fastlane/lib/fastlane/actions/device_grid/device_grid.rb

```diff
@@ -0,0 +1,96 @@
+# rubocop:disable Style/IndentationConsistency
+# We disable this rule since we intend opening HTML labels
+module Danger
+  class Dangerfile
+    module DSL
+      # A danger plugin: https://github.com/danger/danger
+      class DeviceGrid < Plugin
+        # @param languages: Array of languages you want to see (e.g. [en, de])
+        # @param devices: Array of deviecs you want to see (e.g. ["iphone4s", "ipadair"])
+        # @param prefix_command: Prefix the `fastlane run appetize_url_generator` command with something
+        #   this can be used to use `bundle exec`
+        def run(languages: nil, devices: nil, prefix_command: nil)
+          public_key_path = "fastlane/appetize_public_key.txt"
```

Comment:
if this danger plugin is to be reused outside of Fastlane projects, you might want to search for the file under `fastlane/` or `.fastlane/`.